documents/knowledge/elements.yaml

```diff
@@ -310,14 +310,16 @@
   description: Used for list-related digraphs
 
 - element: "ƈ"
-  name: Choose / random choice / set same
-  description: Binomial coefficient / choose a random items from b / same except duplicates
+  name: Choose / random choice / set same / drop while
+  description: Binomial coefficient / choose a random items from b / same except duplicates / drop while
   arity: 2
   overloads:
       num-num: a choose b (binomial coefficient)
       num-str: choose a random items from b
       str-num: choose b random items from a
       str-str: are the set of characters in the strings the same?
+      any-fun: remove items from the beginning of a until b(x) returns false
```

Comment:
What is `x`? The whole array? The dropped item?

Reply:
Each element of the array, I guess

Reply:
How about
```suggestion
      any-fun: remove each item x from the beginning of a until b(x) returns false
```